### ISSUE TITLE
debug / texture issue

### ISSUE THREAD

**Issue by mo8Zomo0** [kovacsv/Online3DViewer#42]
Hi! First of all thx for the project!

I am trying to display a exported project form metashape (export model > wavefront obj with jpg texures) on a self contained web page.
It is just a single model. I have an other model which works fine.
i am using
`<canvas id="fullscreen" class="3dviewer" sourcefiles="3d.jpg|3d.obj|3d.mtl" width="200" height="200"></canvas>`

In the chrome console I can see all files get loaded and I had an other model with the exact
setup which works fine. On this model only the grey shape is visible but the textures do not show up.
When I use the version where I can manually upload the files the model shows fine.

I am using the latest chrome (89.0.4389.114) on the latest macos (11.2.3).

the only error (lets ignore the favicon ;-)  I see on the developer console is 

`content.js:4 [Deprecation] chrome.loadTimes() is deprecated, instead use standardized API: nextHopProtocol in Navigation Timing 2. https://www.chromestatus.com/features/5637885046816768.
(anonymous) @ content.js:4
content.js:5 [Deprecation] chrome.loadTimes() is deprecated, instead use standardized API: nextHopProtocol in Navigation Timing 2. https://www.chromestatus.com/features/5637885046816768.`

but that the same on the one which properly shows.

Any hint what might be wrong or how to debug?

<img width="1575" alt="Bildschirmfoto 2021-04-02 um 13 44 07" src="https://user-images.githubusercontent.com/2004752/113413309-a7011c00-93ba-11eb-8012-4b59fb4730a1.png">

**Comment by kovacsv** [comment]
Hi mo8Zomo0,

From the code snippet it seems that you are using the old version of the library. Have you tried with the current version? Please check if it works with the new version by following the steps in this document:
https://github.com/kovacsv/Online3DViewer/wiki/Use-the-Viewer-on-Your-Website

If it doesn't work it would be a big help if you could share the model and the exact code you use.

**Comment by mo8Zomo0** [comment]
I am not sure how old "old" is, I based it on the github version like 1-2 weeks ago (50723d1).
I used the same base and sample to do the page containing the auto loaded files.
I see there is an update from yesterday and the new one 135e97f

**Comment by mo8Zomo0** [comment]
side question, which type of file set (generated with metashape) would be the preferred one and are there easy ways to scale down the size (last is possibly more a metashape question)

**Comment by kovacsv** [comment]
The code was completely rewritten, and the new version was published one week ago (from version 0.7.0).

**Comment by kovacsv** [comment]
I don't know metashape's capabilites, but if you are looking for optimized size one of the binary formats is recommended. 3ds and glb can handle materials and textures.

**Comment by mo8Zomo0** [comment]
@kovacsv thx!

Hmm, I was just looking at the new samples, is there one for what I am doing here, plain
white background with the auto loaded 3d model and no options like loading other files.

**Comment by kovacsv** [comment]
Here is the documentation for that:
https://github.com/kovacsv/Online3DViewer/wiki/Use-the-Viewer-on-Your-Website

Here is an example fully working page:
https://github.com/kovacsv/Online3DViewer/blob/master/tools/sandbox/embed_selfhost.html

**Comment by mo8Zomo0** [comment]
hmm, that docu and selfhost page includes 	
`<script type="text/javascript" src="../../build/final/website/o3dv/o3dv.min.js"></script>`

I can not find any file o3dv*.js 

```
# find ../.. -iname "o3dv*"
../../3d/Online3DViewer/website/o3dv
```

only the directory 

```
# ls Online3DViewer/website/o3dv
dialogs.js  embed.js  hashhandler.js  info.js  loader.js  menu.js  modal.js  modeldata.js  toolbar.js  treeview.js  utils.js  website.css  website.js
```

**Comment by kovacsv** [comment]
Please read the documentation, and use the files from the release instead of the sandbox:
https://github.com/kovacsv/Online3DViewer/wiki/Use-the-Viewer-on-Your-Website

Downlad the latest version:
https://github.com/kovacsv/Online3DViewer/releases/download/0.7.4/o3dv_0.7.4.zip

Use the files downloaded files:
```html
<script type="text/javascript" src="three.min-126.js"></script>
<script type="text/javascript" src="o3dv.min-0.7.4.js"></script>
```

**Comment by mo8Zomo0** [comment]
ok, though would be handy if they would have been included in the git current git repository?

What I like on the new version is that it tells you about the state so you know whats going on 
(the old one just had a white page till something appeared).

What I noticed on the old one, even with width and hight present it used the full browser page
(which I actually preferred) now it is limiting to the width and height and at least removing these 
did not seem to work at all and a 100% also did not work. 
Is there a way (I am directly accessing the web page) so that the model / canvas scales to the available 
bowser window size? 

I found the error btw, as the new page behaved the same I looked closer at the files and I just saw that the obj and mtl file also contain the filenames. after fixing that it shows up as expected.
(in the new version with the sizing limitations)
